librocksdb-sys/rocksdb_lib_sources.txt

```diff
@@ -166,6 +166,12 @@ options/options_helper.cc
 options/options_parser.cc
 port/mmap.cc
 port/port_posix.cc
+port/win/env_default.cc
+port/win/env_win.cc
+port/win/io_win.cc
+port/win/port_win.cc
+port/win/win_logger.cc
+port/win/win_thread.cc
```

Comment:
The files are needed for win should be skipped here. They are added in the `build.rs` script for win target.